### PR TITLE
Update 3.9.2 changelog and fixes for 4.0 changelog

### PR DESCRIPTION
**Pull request by claudiosanches**
Some updates and fixes for the changelog.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
 
 = 4.0.0 - 2020-03-03 =
 
-* Add - Add woocommerce_admin_process_variation_object hook before $variation->save(), pass WC_Product_Variation object to hooks in the variation metabox. #24929
```
Comment (claudiosanches) [inline readme.txt]: We don't use `Add` in our changelogs, also those are dev related changes, so should use `Dev` prefix.

Comment (claudiosanches) [inline readme.txt]: We also doesn't describe the whole use of the filter, developers can check it in our API docs.

File: readme.txt [hunk 2]
```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
...
-* Dev - Allow filtering of default meta value inside WC_Data::get_meta even if meta key not found. #24066
-* Dev - Includes Emogrifier composer package instead of including into includes/libraries. #25525
-* Dev - Introduce WC_Countries::get_vat_countries for returning a list of countries that uses VAT and refactor WC_Countries::get_european_union_countries with backward compatibility and deprecation to remove the VAT functionality from there. Brexit, remove GB from WC_Countries::get_european_union_countries. #24943
-* Dev - Introduced woocommerce_download_product_filepath filter. #25485
-* Dev - Introduced woocommerce_email_content_type filter. Updated woocommerce_email_from_name filter arguments to include default wp_email() "from" name. Updated woocommerce_email_from_address filter arguments to include default wp_email() email address. #25405
-* Dev - Introduced woocommerce_shortcode_products_query_results filter. #25573
-* Dev - Removed hash_equals() polyfill as it was no longer needed. #25474
-* Dev - Removed unused .order-actions CSS. #25581
 * Enhancement - Included information about packages in the System Status Report. #25584
```
Comment (claudiosanches) [inline readme.txt]: `Enhancement` is always the first section, so people can easily find what is new.

File: readme.txt [hunk 3]
```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
...
 * Enhancement - New WooCommerce Admin. #25011
 * Enhancement - Update dependency woocommerce/woocommerce-blocks to v2.5.12 #25587
 * Enhancement - Updated Action Scheduler to 3.0.1. #25566
+* Performance - Improved the client-side preparation for variation saving. #25382
+* Tweak - Enhance order details and payment summary. #25504
```
Comment (claudiosanches) [inline readme.txt]: `Tweak` session comes after `Enhancement`.

File: readme.txt [hunk 4]
```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
...
+* Tweak - Increased range and precision for `min_price` and `max_price` in the product meta lookup table. #25253
+* Tweak - Move action scheduler to external via composer. #25404
+* Tweak - Only update the customer IP address when order gets created from admin. #25137
+* Tweak - Remove WooCommerce Admin install alert. #25502
+* Tweak - Removed WC Admin from the Setup Wizard if it's already active. #25562
+* Tweak - Set email header table width to 100% for full width. #25294
+* Tweak - Simplified MaxMind integration title. #25522
+* Tweak - Update 'Country' to 'Country / Region' label. #25530
+* Tweak - WooCommerce.com plugins auto-install and update improvement. #25532
 * Fix - Add missing closing tag. #25319
```
Comment (claudiosanches) [inline readme.txt]: And now finally the list of Fixes followed with Dev related changes.

File: readme.txt [hunk 5]
```diff
@@ -213,25 +209,30 @@ INTERESTED IN DEVELOPMENT?
 * Fix - Fixed get_discount function return type. #25567
 * Fix - Fixed possible multiple is_vat_exempt order meta in order creation. #25426
 * Fix - Fixed wrong context help for translators. #25496
-* Fix - Make WC_Shipping::is package shippable less strict. #25386
-* Fix - Prevent undefined wp_delete_user() error while removing inactive accounts. #25489
+* Fix - Make `WC_Shipping::is` package shippable less strict. #25386
```
Comment (claudiosanches) [inline readme.txt]: Note that we indicate code with markdown.

**Comment by peterfabian** [comment]
Thanks for the updates, I ordered the changelog alphabetically. 🤷‍♂️

**Review by peterfabian — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**